test/lender/Lender.sol

```diff
@@ -26,6 +26,8 @@ contract Lender {
     address public admin;
     /// @notice address of the MarketPlace.sol contract, used to access the markets mapping
     address public marketPlace;
+    /// @notice mapping that determines if a market may be used by a lender
+    mapping(address => mapping(uint256 => bool)) public paused;
```

Comment:
mb not describing better, should map the principal enum to paused rather than u,m

This lets us pause a protocol if it gets hacked, rather than a whole market

Reply:
No worries, just fixed it!